tests/Feature/SubmissionValidation.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\Project;
+use Exception;
+use Tests\TestCase;
+use Tests\Traits\CreatesProjects;
+use Tests\Traits\CreatesSubmissions;
+
+class SubmissionValidation extends TestCase
+{
+    use CreatesSubmissions;
+    use CreatesProjects;
+
+    protected string $ConfigFile = '';
+    protected mixed $Original = '';
+    protected Project $project;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->ConfigFile = dirname(__FILE__) . '/../../.env';
```

Comment:
Same comment here: can we use `base_path()` for this?

Reply:
Done in both cases.  Thanks!